bin/setup

```diff
@@ -34,6 +34,8 @@ Dir.chdir APP_ROOT do
   run "cp keys/equifax_rsa.example keys/equifax_rsa"
   run "cp keys/equifax_rsa.pub.example keys/equifax_rsa.pub"
   run "cp certs/saml.crt.example certs/saml.crt"
+  run "gpg --dearmor < keys/equifax_gpg.pub.example > keys/equifax_gpg.pub.bin"
+  system "gpg --batch --import keys/equifax_gpg.example"
```

Comment:
my hesitancy here is polluting the local user's gpg keyring. That's why I had originally used
```
--keyring #{Shellwords.shellescape(gpg_key_path)} \
```
in the encryption command, to avoid importing the key.